tools/diffacto/diffacto.xml

```diff
@@ -0,0 +1,230 @@
+               If None, read from peptide file headings, 
+               then each run will be summarized as a group.
+            ]]></help>
+        </param>
+        <param argument="-reference" type="text" value="" label="reference sample groups" 
+            help="Names of reference sample groups (separated by semicolon)">
+        </param>
+        <param name="scale" argument="-log2" type="select" label="Peptides abundance scale">
+            <option value="linear">linear</option>
+            <option value="log2">log2</option>
+        </param>
+        <param argument="-normalize" type="select" label="Method for sample-wise normalization" optional="true">
+            <option value="average">average</option>
+            <option value="median">median</option>
+            <option value="GMM">GMM</option>
+        </param>
+        <param argument="-farms_mu" type="float" value="0.1" min="0.0" max="1.0" optional="true" label="Hyperparameter mu"
+            help="Hyperparameter mu (default: 0.1)"/>
+        <param argument="-farms_alpha" type="float" value="0.1" min="0.0" max="1.0" optional="true" label="Hyperparameter alpha"
+            help="Hyperparameter weight of prior probability (default: 0.1)"/>
+        <param argument="-min_samples" type="integer" value="1" min="1" label="Minimum samples for peptide"
+            help="Minimum number of samples peptides needed to be quantified in"/>
+        <param argument="-impute_threshold" type="float" value="0.99" min="0.1" max="1.0" label="Minimum fraction of missing values in the group"
+            help="Impute missing values if missing fraction is larger than the threshold."/>
+        <param argument="-cutoff_weight" type="float" value="0.5" min="0." max="1.0" label="Peptide cutoff weight"
+            help="Peptides weighted lower than the cutoff will be excluded."/>
+        <param argument="-use_unique" type="boolean" truevalue="-use_unique True" falsevalue="" checked="false" label="Use unique peptides only"/>
+        <param argument="-fast" type="boolean" truevalue="-fast True" falsevalue="" checked="false" label="Allow early termination in EM calculation when noise is sufficiently small."/>
+        <param name="mcfdr" argument="-mc_out" type="boolean" truevalue="True" falsevalue="False" checked="false" label="Perform Monte Carlo FDR simulation"/>
+        <param name="loadings" argument="-loadings_out" type="boolean" truevalue="True" falsevalue="False" checked="false" label="Output Protein Peptide loadings file"/>
+    </inputs>
+    <outputs>
+        <data name="output" format="tabular" label="${input.name.replace('.csv','')}.tsv">
+            <actions>
+                <action name="comment_lines" type="metadata" default="1" />
+                <action name="column_names" type="metadata" default="Protein,N.Pept,Q.Pept,S/N,P(PECA)" />
+            </actions>
+        </data>
+        <data name="mc_out" format="tabular" label="${input.name.replace('.csv','')}.FDR">
```

Comment:
users can change that name as they like.

What about: `${tool.name} on ${on_string}: FDR` or `${tool.name} on ${on_string}: Tabular` above.

Reply:
@bgruening I reverted to the standard method you suggested.